src/reducers.js

```diff
@@ -5,12 +5,10 @@
 
 import { combineReducers } from 'redux';
 
-import { reducer as navigation } from './navigation';
-import { reducer as sync } from './sync';
+import { reducer as appReducer } from './navigation';
+import { reducer as syncReducer } from './sync';
 
-export const reducers = combineReducers({
-  navigation,
-  sync,
+export default combineReducers({
+  nav: appReducer,
```

Comment:
are we able to return this to navigation again? Then we can stop using nav everywhere?

Reply:
@joshxg navigation is used by react navigation by default. Because we are using redux, we are avoiding react navigation's navigation variable and setup our own. If we use nav the variable names will conflict.

I would have preferred calling it navigation as well.